src/main/java/core/basesyntax/FigureSuplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSuplier {
+    private static final int MAX_RANDOM_NUMBER = 50;
+    private static final int DEFAULT_CIRCLE_RADIUS = 10;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int index = random.nextInt(FigureType.values().length);
+        FigureType randomFigure = FigureType.values()[index];
+
+        switch (randomFigure) {
+            case SQUARE:
+                return new Square(colorSupplier.getRandomColor(),
+                        random.nextDouble() * MAX_RANDOM_NUMBER);
```

Comment:
```suggestion
                        MAX_RANDOM_NUMBER);
```
the same in all other places below

Reply:
Its not clear to me, according to task we should generate figure with random parameters. If i remove random, i will use constant (50 in my case).

Reply:
> Its not clear to me, according to task we should generate figure with random parameters. If i remove random, i will use constant (50 in my case).